packages/@coorpacademy-components/locales/en/global.json

```
@@ -20,6 +20,7 @@
   "Create a new cohort": "Create a new cohort",
   "Delete": "Delete",
   "Draft": "Draft",
+  "drop_down_icon": "Dropdown icon, to open a list of available answers",
```

```suggestion
  "drop_down_icon": "Dropdown to open the list of available answers",
```
le user sais déjà que c'est un svg

---

@audric-coorp